openedx/core/djangoapps/verified_track_content/management/commands/swap_from_auto_track_cohort_pilot.py

```diff
@@ -170,6 +170,7 @@ def handle(self, *args, **options):
                     )
                     if (audit_partition_group_access
                             and audit_course_user_group_partition_group.group_id in audit_partition_group_access):
+                        print "Queueing XBlock at location: '%s' for Audit Content Group update " % item.location
```

Comment:
No specific reason other than I knew that printing to stdout will be directed to the Jenkins console, I'd swap it but I'd rather not have to re-verify that log.info behaves in the same way.